src/NuGet.Clients/NuGet.VisualStudio.Common/ErrorListTableEntry.cs

```diff
@@ -14,7 +14,7 @@ namespace NuGet.VisualStudio.Common
     public class ErrorListTableEntry : ITableEntry
     {
         internal const string ErrorSouce = "NuGet";
-        internal const string HelpLink = "https://msdn.microsoft.com/query/dev15.query?appId=Dev15IDEF1&l={0}&k=k({1})&rd=true";
+        internal const string HelpLink = "https://msdn.microsoft.com/query/dev16.query?appId=Dev16IDEF1&l={0}&k=k({1})&rd=true";
```

Comment:
I think `https://docs.microsoft.com/nuget/reference/errors-and-warnings/{1}` or `https://docs.microsoft.com/{0}/nuget/reference/errors-and-warnings/{1}` would be a better pattern, since that's the actual URL now.

Reply:
@zivkan There is one issue with that. If an error was added but still not documented, the link will go to 404 page. Using the msdn query link will redirect to some docs landing page.

I'll let you decide, as you for sure will have more knowledge about how common having a new undocumented error.

Reply:
@zivkan Also, keep in mind this issue (https://github.com/NuGet/docs.microsoft.com-nuget/issues/2143)

Reply:
That's a great point. However I'd argue that the docs site 404 is a better experience.

If someone clicks a link to get help on a specific NU error code and gets redirected to some generic Visual Studio page, I think they'd be confused and not sure where to go. How does that page help them with the error they got in the error list?

Whereas the docs site 404 is a more clear that whatever NuGet intended to show the customer is not working correctly. Plus it has NuGet specific search results. In my tests of using non-existant NU codes, the "NuGet errors and warnings reference" was one of the links.

If you docs issue reporting these missing codes was reported on that page, it should have appeared at the bottom of the page, which would have made it easy for the customer to see what we're missing docs. And in any case if the customer clicks one of those links to a working NuGet docs page, they'll see ways to provide feedback about the bad link. With the redirection to the generic VS product page, it's not easy for the customer to report the problem to us.

At least, that's my opinion.